config/initializers/blacklight_initializer.rb

```diff
@@ -65,7 +66,14 @@ def export_as_openurl_ctx_kev(format = nil)
         export_text << "&amp;rft.isbn=#{(isbn.nil? or isbn['a'].nil?) ? "" : isbn['a']}"
       end
     end
-    export_text << "&amp;rft_id=" + (id.nil? ? "" : CGI::escape("pul_"+id.value))
+    
+    export_text << "&amp;rft_id=" + (id.nil? ? "" : CGI::escape("http://bibdata.princeton.edu/bibliographic/"+id.value))
```

Comment:
How does the rft_id value get used?
Stylewise I think it's more readable to interpolate the string: `"http://bibdata.princeton.edu/bibliographic/{id.value}"`